ml/Config.cc

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#include "Config.h"
+#include "ml-private.h"
+
+using namespace ml;
+
+/*
+ * Global configuration instance to be shared between training and
+ * prediction threads.
+ */
+Config ml::Cfg;
+
+/*
+ * Initialize global configuration variable.
+ */
+void Config::readMLConfig(void) {
+    const char *ConfigSectionML = "ml";
+
+    Cfg.TrainSecs = Seconds{config_get_number(ConfigSectionML, "num secs to train", 4 * 60)};
+    Cfg.MinTrainSecs = Seconds{config_get_number(ConfigSectionML, "minimum num secs to train", 1 * 60)};
+    Cfg.TrainEvery = Seconds{config_get_number(ConfigSectionML, "train every secs", 1 * 60 )};
+
+    Cfg.DiffN = config_get_number(ConfigSectionML, "num samples to diff", 1);
+    Cfg.SmoothN = config_get_number(ConfigSectionML, "num samples to smooth", 3);
+    Cfg.LagN = config_get_number(ConfigSectionML, "num samples to lag", 5);
+
+    Cfg.DimensionAnomalyScoreThreshold = config_get_float(ConfigSectionML, "dimension anomaly score threshold", 0.99);
+    Cfg.HostAnomalyRateThreshold = config_get_float(ConfigSectionML, "host anomaly rate threshold", 0.2);
+
+    Cfg.ADWindowSize = config_get_float(ConfigSectionML, "window minimum size", 30);
+    Cfg.ADWindowRateThreshold = config_get_float(ConfigSectionML, "window minimum anomaly rate", 0.25);
+    Cfg.ADDimensionRateThreshold = config_get_float(ConfigSectionML, "anomaly event min dimension rate threshold", 0.1);
+
+    std::string HostsToSkip = config_get(ConfigSectionML, "hosts to skip from training", "!*");
+    Cfg.SP_HostsToSkip = simple_pattern_create(HostsToSkip.c_str(), NULL, SIMPLE_PATTERN_EXACT);
+
+    std::string ChartsToSkip = config_get(ConfigSectionML, "charts to skip from training", "!system.cpu *");
```

Comment:
```suggestion
    std::string ChartsToSkip = config_get(ConfigSectionML, "charts to skip from training", "!system.* !cpu.* !mem.* !disk.* !disk_* !ip.* !ipv4.* !ipv6.* !net.* !net_* !netfilter.* !services.* !apps.* !groups.* !user.* !ebpf.* !netdata.* *");
```

Reply:
Hey! A question: Could it be a bit confusing as to what `charts to skip from training` (and the `hosts to skip from training` config) mean along with the `!` notation?

If I put `charts to skip from training = !system.*` does this mean `system.*` charts are skipped, or included?

Perhaps if the config option would be `charts to include in training` would be more clear?

Reply:
@andrewm4894 what do you think about @MrZammler suggestion?

Reply:
i agree it is very confusing. i thought or assumed was something special relating to [netdata simple patterns](https://learn.netdata.cloud/docs/agent/libnetdata/simple_pattern) that just made this way easier so kept quiet :)

but yes, yes it is confusing for a user and using `charts to include` and `hosts to include` would be better if not a complex thing to change. 

So i think:

`hosts to skip from training` would be better as `hosts to include`
`charts to skip from training` would be better as `charts to include`

We don't need the `in training` part as in reality it either includes for both training and prediction so simpler to just say `include`.

Reply:
Will post a follow-up PR for this, given that this is not a request for a functional change.